### PR TITLE
[improvement](auth) support show view priv

### PR DESCRIPTION
**Pull request by zddr**
## Proposed changes

Issue Number: close #xxx

current ,if user has `select_priv` or `load_priv`,he can `show create table view_name`,
but this is not safe，so add `show_view_priv` for `show create table view_name`

mysql `SHOW VIEW`  description: https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_show-view

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run p0

**Comment by zddr** [comment]
run buildall

**Comment by zddr** [comment]
run buildall

File: docs/zh-CN/docs/sql-manual/sql-reference/Account-Management-Statements/GRANT.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ privilege_list 是需要赋予的权限列表，以逗号分隔。当前 Doris 
     CREATE_PRIV：对指定的库或表的创建权限
     DROP_PRIV：对指定的库或表的删除权限
     USAGE_PRIV: 对指定资源的使用权限<version since="dev" type="inline" >和workload group权限</version>
+    SHOW_VIEW_PRIV: 查看`view`创建语句的权限(从2.0.3版本开始，`SELECT_PRIV`和`LOAD_PRIV`权限不能`SHOW CREATE TABLE view_name`)
```
Comment (morningman) [inline docs/zh-CN/docs/sql-manual/sql-reference/Account-Management-Statements/GRANT.md:65]: 这里要再补充下完整的，即用户拥有哪些权限时，可以执行show create

**Comment by zddr** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
lgtm

**Pull request merged**